pajbot/managers/command.py

```diff
@@ -440,6 +448,7 @@ def parse_command_arguments(
         parser.add_argument("--tokens-cost", type=int, dest="tokens_cost")
         parser.add_argument("--modonly", dest="mod_only", action="store_true")
         parser.add_argument("--no-modonly", dest="mod_only", action="store_false")
+        parser.add_argument("--enabled", type=str, dest="command_state")
```

Comment:
There's already `--enable` below (line 457.) which is very similar to `enabled`. Perhaps this argument should be called something different?

On another note, this should copy the behavior from e.g. `--modonly` / `--no-modonly`, so this line should look a bit different and have another argument with `action="store_false"`
```suggestion
        parser.add_argument("--enabled", dest="command_state", action="store_true")
```